examples/libs/ripple/basic-tx/main.ts

```diff
@@ -0,0 +1,60 @@
+import { DfnsWallet } from '@dfns/lib-ripple'
+import { DfnsApiClient } from '@dfns/sdk'
+import { AsymmetricKeySigner } from '@dfns/sdk-keysigner'
+
+import * as dotenv from 'dotenv';
+import { Client, Transaction, encode } from 'xrpl';
+
+dotenv.config()
+
+const initDfnsWallet = async (walletId: string) => {
+  const signer = new AsymmetricKeySigner({
+    privateKey: process.env.DFNS_PRIVATE_KEY!,
+    credId: process.env.DFNS_CRED_ID!,
+    appOrigin: process.env.DFNS_APP_ORIGIN!,
+  })
+
+  const dfnsClient = new DfnsApiClient({
+    appId: process.env.DFNS_APP_ID!,
+    authToken: process.env.DFNS_AUTH_TOKEN!,
+    baseUrl: process.env.DFNS_API_URL!,
+    signer,
+  })
+
+  return DfnsWallet.init({
+    walletId: walletId,
+    dfnsClient,
+  })
+}
+
+async function main() {
+  const senderWallet = await initDfnsWallet(process.env.RIPPLE_WALLET_ID!)
+  console.log("ripple sender address: %s", senderWallet.address)
+  
+  let tx: Transaction = {
+    TransactionType: 'Payment',
+    Account: senderWallet.address,
+    // Replace with your desired receiver
+    Destination: "rBYtCQKxGTfFuob3hxSc8pEYddetT9CdDZ",
+    Amount: '100000',
+  }
+
+  // Choose the client you want.
+  const client = new Client('wss://testnet.xrpl-labs.com')
+  
+  // Connect the websocket client
+  await client.connect().then(async () => {
```

Comment:
don't use `then ... finally`. that's the old promise style before `async/await`.

also, see the other comment, we can simplify this example down to 

```
const res = await client.submitAndWait({
    TransactionType: 'Payment',
    Account: wallet.address,
    Destination: "rBYtCQKxGTfFuob3hxSc8pEYddetT9CdDZ",
    Amount: '1',
  },
  {
    autofill: true,
    wallet,
  })
```

Reply:
I miss my 'defer' statement in go :)

Reply:
it's just a simple example, don't really care about error handling. all the examples we just assume they will work.